pkg/pool/operations/pool_utils.go

```diff
@@ -295,3 +313,39 @@ func (oc *OperationsConfig) cleanUpReplacementMarks(oldObj, newObj *openebsapis.
 	}
 	return nil
 }
+
+// SetCompression sets the cstor pool compression
+func SetCompression(poolName string, compressionType string) error {
+	// If compression type is empty -- it means disable compression on the pool
```

Comment:
let us act only when user has given something. Let us not do anything if compressionType is empty.

Reply:
Empty is an invalid value -- so in this case, will return an error.
Initially, I thought that empty means off so did like this.
Thanks.

Reply:
what if user has not added compression in the CR, what you will get here?

Reply:
It will be empty what does it mean?

Reply:
then we should not treat that as error. And we should not assume anything also. We should fire the command without the comression property.

Reply:
Makes sense. Will do that.

Reply:
Thanks Pawan

Reply:
When I looked back, I think the current implementation looks good to me. 
Happy to discuss more.